phoenix-core/src/main/java/org/apache/phoenix/parse/IndexExpressionParseNodeRewriter.java

```diff
@@ -51,6 +51,12 @@ public Void visit(ColumnParseNode node) throws SQLException {
             return null;
         }
         
+        @Override
+        public boolean visitEnter(FunctionParseNode node) throws SQLException {
+            isParseNodeCaseSensitive = true;
```

Comment:
Is there a downside to always surrounding column names with double quotes when we run the IndexExpressionParseNodeRewriter?